rust/sdk/Cargo.toml

```diff
@@ -38,46 +41,57 @@ serde_cbor = "0.11.2"
 url = "2.5.0"
 tracing = "0.1.40"
 
-# This feature provides exponential backoff on requests
-tokio = { version = "1.34.0", optional = true }
+# Feature: with_tokio
+tokio = { version = "1.15.0", features = [
```

Comment:
what's the reason for downgrading 1.34.0 to 1.15.0? (that is if I'm reading this correctly)